build-system/babel-config/jss-config.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+const options = {css: 'REPLACED_BY_BABEL'};
```

Comment:
Maybe add a comment saying what this is for?

Also, instead of renaming this from `options` to `jssOptions` in `extension-helpers.js`, how about calling it `jssOptions` here so there's no chance of confusion during any future usage?

Reply:
Sure